src/contracts/mixins/GPv2Signing.sol

```diff
@@ -81,6 +85,7 @@ abstract contract GPv2Signing {
         } else {
             preSignature[orderUid] = 0;
         }
+        emit PreSignature(owner, orderUid, signed);
```

Comment:
No strong opinion at all but just wondering if you thought about emitting two events, one for signing and one for revoking a signature?

Reply:
No I didn't really think about it to be honest... I'm not sure which I prefer either :stuck_out_tongue: